Sources/Extensions/CustomTypes/AnalyticsService+GDS.swift

```diff
@@ -8,6 +8,7 @@ enum AppTaxonomy: String {
     case home
     case wallet
     case profile
+    case reauth = "re auth"
```

Comment:
should this have a space in it? maybe something we need to mention to the analytics team to see if we can avoid

Reply:
For this comment and the one about "my profile", I've started a thread with the analytics team on [Slack](https://gds.slack.com/archives/C068SNY9UFP/p1726650547125459)